Alicerce/Sources/Extensions/Thread.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Thread.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 17/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+extension Thread {
+
+    // Returns the name of the thread or 'main-thread', if it's the application's main thread
+    public class func threadName() -> String {
```

Comment:
I think this is a good candidate for a computed property, for the following reasons:
1) There is no major computation taking place that the caller needs to be aware of
2) I would expect the thread name to be a simple string property, not a function
3) It looks cleaner at the call site

:smile: